custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -31,14 +31,15 @@ class PredictMixin:
 
     def do_predict(self, logger=None):
         response_status = HTTP_200_OK
-        response = None
 
         file_key = "X"
         filestorage = request.files.get(file_key)
 
         if not filestorage:
             wrong_key_error_message = (
-                "Samples should be provided as a csv file under `{}` key.".format(file_key)
+                "Samples should be provided as a csv, mtx, or arrow file under `{}` key.".format(
```

Comment:
this is tricky, as not all predictors support Arrow right now (only python ones)
same for mtx
the only common format is csv. other formats can checked via /capabilities/ endpoint.
I think we should say that